docs/adr/0036-payments-standalone-configuration.md

```diff
@@ -0,0 +1,92 @@
+# Refactor Subscription Platform frontend
```

Comment:
Oh yeah, is this an accurate way to reflect the content of this ADR? Seems less about the front-end and more about how to manage product/plan config on the back-end.